pages/tutorials/index.html

```diff
@@ -33,52 +37,58 @@ <h1 class="display-4">Getting started with Cantera</h1>
     <div class="card-deck">
       <div class="card">
         <div class="card-body">
-          <h1 class="card-title">I want to learn about Cantera via the Python module.</h1>
+          <h1 class="card-title">I want to learn about Cantera via the Python
+            module.</h1>
         </div>
         <div class="card-footer">
-          <a href="../cython/tutorial.html" class="btn btn-primary">Python Tutorial</a>
+          <a href="PythonTutorial.html" class="btn btn-primary">Python Tutorial</a>
         </div>
       </div>
       <div class="card">
         <div class="card-body">
-          <h1 class="card-title">I want to learn about Cantera via the Matlab toolbox.</h1>
+          <h1 class="card-title">I want to learn about Cantera via the Matlab
+            toolbox.</h1>
         </div>
         <div class="card-footer">
-          <a href="../matlab/tutorial.html" class="btn btn-primary">Matlab Tutorial</a>
+          <a href="MatlabTutorial.html" class="btn btn-primary">Matlab Tutorial</a>
```

Comment:
The links here need to be to the slug, i.e., `matlab-tutorial.html`, except your slug line isn't right, it should be `.. slug: matlab-tutorial` (only one `:` and a space before the `matlab-tutorial`. Same for Python. If you'd rather keep it this way, URLs are case-insensitive, so these end up being all lower-case file names (at least on my machine) although everything works out because of the case-insensitivity.